pkg/kubecost/asset.go

```diff
@@ -2838,6 +2838,31 @@ func (as *AssetSet) accumulate(that *AssetSet) (*AssetSet, error) {
 	return acc, nil
 }
 
+type Diff struct {
+	Asset Asset
+	ChangedType string
+}
+
+func DiffAsset(before, after *AssetSet) []Diff{
+	changedAssets := []Diff{}
+
+	for assetKey1, asset1 := range before.assets {
+		if _, ok := after.assets[assetKey1]; !ok {
+			d := Diff{asset1, "removed"}
```

Comment:
What do you think about codifying these strings as constants?

Reply:
This is definitely a good idea, I'll get this added.